e2e/testcases/hydration_test.go

```diff
@@ -118,102 +118,6 @@ func TestHydrateKustomizeComponents(t *testing.T) {
 	}
 }
 
-func TestHydrateHelmComponents(t *testing.T) {
```

Comment:
Another option is to use `t.Skip()` so that the test still shows up in the test reports as skipped. Also would make this diff a bit smaller. I'm okay with either option

Reply:
I thought about `t.Skip()` but given that it's a release branch and it's a test we no longer need. I just removed it.